spec/rmagick/gradient_fill/fill_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe Magick::GradientFill, '#fill' do
+  it 'works' do
+    img = Magick::Image.new(10, 10)
+
+    expect do
```

Comment:
Mayb you could split this up in multiple tests in this PR?

Reply:
Hmm, these tests don't actually seem to be testing anything distinct between them. They're just testing that `fill` always returns the `gradient` with different arguments. Not sure, but I don't think the return value is even something we should care about here. It seems like when we're ready to flesh this test out, we should be testing that the pixels in `img` have somehow changed.